README.md

```diff
@@ -32,6 +33,33 @@ Read the [Changelog](https://github.com/Microsoft/vscode-go/blob/master/CHANGELO
 - [Code of Conduct](#code-of-conduct)
 - [License](#license)
 
+## We are moving!
+
+🏡 🚚 🏡 
+
+Our new home will be https://github.com/golang/vscode-go.
+
+As the extension grows in popularity and supports more features, it has required more maintenance and support.
```

Comment:
Thanks for the edits, @ramya-rao-a! I like this paragraph, but I have a few small edits (new version below):

As the extension grows in popularity and supports more features, it requires more maintenance and support. Since the introduction of Go modules, we have been collaborating with the Go team for module support. In 2019, we added support for `gopls`, the [Go language server](https://golang.org/s/gopls/README.md), which has resulted in closer collaboration with the Go team. More exciting work is on the horizon with the
[Go debug adapter service](https://github.com/go-delve/delve/tree/master/service/dap) in Delve, which will help improve the debugging experience.

Reply:
Fixed in https://github.com/microsoft/vscode-go/pull/3237/commits/b9071ad8d7037db5b8c33790ea969580de42f3d1